examples/.github/workflows/release-on-milestone-closed.yml

```diff
@@ -35,3 +35,13 @@ jobs:
           "SIGNING_SECRET_KEY": ${{ secrets.SIGNING_SECRET_KEY }}
           "GIT_AUTHOR_NAME": ${{ secrets.GIT_AUTHOR_NAME }}
           "GIT_AUTHOR_EMAIL": ${{ secrets.GIT_AUTHOR_EMAIL }}
+
+      - name: "Create and/or Switch to new Release Branch"
+        uses: "laminas/automatic-releases@v1"
+        with:
+          command-name: "laminas:automatic-releases:switch-default-branch-to-next-minor"
+        env:
+          "GITHUB_TOKEN": ${{ secrets.GITHUB_TOKEN }}
```

Comment:
As discussed in https://github.com/laminas/automatic-releases/pull/12#issuecomment-660946161, this will need a separate token

Reply:
This now uses a completely independent environment variable